src/language/templates/visitors/checkparent_visitor.cpp

```diff
@@ -0,0 +1,64 @@
+/*************************************************************************
+ * Copyright (C) 2018-2019 Blue Brain Project
+ *
+ * This file is part of NMODL distributed under the terms of the GNU
+ * Lesser General Public License. See top-level LICENSE file for details.
+ *************************************************************************/
+
+#include <sstream>
+#include <string>
+#include <fmt/format.h>
+#include "visitors/checkparent_visitor.hpp"
+
+namespace nmodl {
+namespace visitor {
```

Comment:
maybe worth introducing an additional namespace `test` for this?

Reply:
@pramodk what do you think?

Reply:
Would be cleaner...up to you to decide if you want to do in same PR or separate PR.